libbeat/processors/add_cloud_metadata/provider_alibaba_cloud.go

```diff
@@ -0,0 +1,41 @@
+package add_cloud_metadata
+
+import "github.com/elastic/beats/libbeat/common"
+
+// Alibaba Cloud Metadata Service
+// Document https://help.aliyun.com/knowledge_detail/49122.html
+func newAlibabaCloudMetadataFetcher(c common.Config) (*metadataFetcher, error) {
+	ecsMetadataHost := "100.100.100.200"
```

Comment:
It's interesting that this is a fixed IP and not a domain name. Couldn't make much sense of the docs above unfortunately ;-)